### PR TITLE
Update gossipping girls total

### PR DESCRIPTION
**Pull request by MathiasMehl**
Adds a few formulas to gossipping girls total

### CODE REVIEW

File: docs/lcgs/working-examples/Gossipping_Girls_Total_Network/p1_omniscient_before_10_steps_TRUE.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "enforce eventually": {
+    "players": [0,1,2,3],
+    "formula" : {
+      "proposition" :  1}
+  }
+}
+
```
Comment (NicEastvillage) [inline docs/lcgs/working-examples/Gossipping_Girls_Total_Network/p1_omniscient_before_10_steps_TRUE.json:8]: Filens navn virker forkert.

Comment (GeneralDenmark) [reply docs/lcgs/working-examples/Gossipping_Girls_Total_Network/p1_omniscient_before_10_steps_TRUE.json]: Fixed in afb042d

File: docs/lcgs/working-examples/Gossipping_Girls_Circular/can_guarantee_all_girls_eventually_become_omniscient_but_not_girl_one.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+              "not": {
+                "propersition": 3
+              }
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
```
Comment (NicEastvillage) [inline docs/lcgs/working-examples/Gossipping_Girls_Circular/can_guarantee_all_girls_eventually_become_omniscient_but_not_girl_one.json:31]: `formula` i stedet for `pre`. Og du tjekker både "proposition 3" og "not proposition 3"

Comment (MathiasMehl) [reply docs/lcgs/working-examples/Gossipping_Girls_Circular/can_guarantee_all_girls_eventually_become_omniscient_but_not_girl_one.json]: fixd in 5a1c726c790f7287e6fe4b586f28e54c15294b73 and 75847f436a68a84228a0f2f4cd1f8389e8126d66

**Review by NicEastvillage — APPROVED** [review]

**Review by GeneralDenmark — APPROVED** [review]

**Pull request merged**